libs/ts4wing/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import { std, platform } from "@winglang/sdk";
+import { Construct } from "constructs";
+
+/**
+ * Properties for a Wing app.
+ */
+export interface WingAppProps {
+  /**
+   * The name and id of the app.
+   * @default "main"
+   */
+  name?: string;
+}
+
+/**
+ * Create a Wing app.
+ *
+ * ```ts
+ * import { wing } from "ts4wing";
+ * import { cloud } from "@winglang/sdk";
+ *
+ * wing(app => {
+ *   new cloud.Bucket(app, "Bucket");
+ * });
+ * ```
+ *
+ * @param fn Define your application using the provided root construct.
+ *           Note that this function may be called multiple times when used with `wing test`.
+ */
+export function wing(fn: (root: Construct) => void, props: WingAppProps = {}) {
```

Comment:
I considered calling this `app`, but I was concerned that people would expect it to be a construct rather than a function. Also I think people may want to call the root construct `app`

Reply:
How about `main`?

```js
main(app => {

  // ...

});
```